src/cli/cli.cpp

```diff
@@ -837,8 +836,6 @@ otError Interpreter::ProcessBackboneRouterMgmtMlr(Arg aArgs[])
          * bbr mgmt mlr listener clear
          * Done
          * @endcode
-         * @par
-         * @note Only for reference and for testing the Backbone Router device.
```

Comment:
For this, looks like both OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE and OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE need to be enabled.  Should we note both rather than removing the note entirely?  @abtink